test/project_types/script/forms/ask_app_test.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require "project_types/script/test_helper"
+
+describe Script::Forms::AskApp do
+  include TestHelpers::FakeFS
+  describe ".ask" do
+    let(:context) { TestHelpers::FakeContext.new(root: Dir.mktmpdir) }
+    let(:acting_as_shopify_organization) { false }
+    let(:selected_api_key) { "default_api_key" }
+    let(:selected_secret) { "default_secret" }
+
+    subject do
+      result = nil
+      capture_io do
+        result = Script::Forms::AskApp.ask(
+          context,
+          {
+            apps: apps,
+            acting_as_shopify_organization: acting_as_shopify_organization,
+          },
+          {}
+        )
+      end
+      result
+    end
+
+    def new_app(title, api_key, secret, app_type = "custom")
```

Comment:
Nitpick: in this case, it's better to not have default value for `app_type`.  It's not obvious that when this parameter is missing that this is creating a custom app.  IMO, the 'T' in 'TRUE' is not satisfied.  Just my 2 cents, not asking for any changes.

Reply:
Fair enough, I'll leave it as is for now since it simplifies the creation of the apps.